backend/src/api/model/series.rs

```diff
@@ -149,6 +149,21 @@ impl Series {
     async fn events(&self, order: EventSortOrder, context: &Context) -> ApiResult<Vec<AuthorizedEvent>> {
         AuthorizedEvent::load_for_series(self.key, order, context).await
     }
+
+    /// Returns `true` if the realm has a series block with this series.
+    /// Otherwise, `false` is returned.
+    pub(crate) async fn is_referenced_by_realm(&self, path: String, context: &Context) -> ApiResult<bool> {
+        let query = "select exists(\
+            select 1 \
+            from blocks \
+            join realms on blocks.realm = realms.id \
+            where realms.full_path = $1 and blocks.series = $2 \
+        )";
```

Comment:
I just noticed something: this, `Series.hostRealms` and the `search_series` DB view all agree on the semantics. However, [our documentation](https://elan-ev.github.io/tobira/semantics#listed-and-being-findable-via-search) says something different: 

> a series is included in a content page, if a series block refers to it or if a video block refers to any of its videos.

I think the documentation was written before series in search was implemented. 

This is completely unrelated to this PR but it brings up the question of what we want. Mhhh